csm/core/agent/csm_agent.py

```diff
@@ -261,6 +266,8 @@ def run():
         if not (env_type == const.K8S):
             CsmAgent.alert_monitor.stop()
             Log.info("Finished stopping alert monitor service")
+        Log.info("Stopping Message Bus client")
+        CsmRestApi._app["comm_client"].stop()
```

Comment:
Call CsmRestApi._app["stat_service"].stop() instead of CsmRestApi._app["comm_client"].stop()

Reply:
Modifications done